app/views/landing_page/blocks/_main_navigation.html.erb

```diff
@@ -1,45 +1,46 @@
 <%
   add_view_stylesheet("landing_page/main-navigation")
 %>
+<div class="app-b-main-nav--border">
+  <div class="app-b-main-nav govuk-width-container" data-module="app-b-main-navigation">
```

Comment:
I'd also put the `data-module` on the parent, if that's not going to cause a problem.